scripts/run_gce_system_tests.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import unittest
+
+import httplib2
+from six.moves import http_client
+
+from oauth2client.client import GoogleCredentials
+from oauth2client.contrib.gce import AppAssertionCredentials
+
+
+SCOPE = ('https://www.googleapis.com/auth/plus.login',
```

Comment:
Copy-pasta. Nice find. (I want #212 to be done so we can turn on `pylint` and 100% coverage so the machines can yell at the humans.)